odoo/addons/base/models/ir_module.py

```diff
@@ -434,14 +434,6 @@ def button_immediate_install(self):
         :rtype: dict[str, object]
         """
         _logger.info('User #%d triggered module installation', self.env.uid)
-        # We use here the request object (which is thread-local) as a kind of
-        # "global" env because the env is not usable in the following use case.
-        # When installing a Chart of Account, I would like to send the
-        # allowed companies to configure it on the correct company.
-        # Otherwise, the SUPERUSER won't be aware of that and will try to
-        # configure the CoA on his own company, which makes no sense.
-        if request:
-            request.allowed_company_ids = self.env.companies.ids
```

Comment:
This one is less clear to me.  How does it have a bad impact?

Reply:
Hi @jco-odoo . It does not have a bad impact. We delete this part since we review the code and look that is not been used anymore. It was used only in the part of loading the chart of account that we delete. Would you like us to revert this change?